app/src/main/java/app/attestation/auditor/RemoteVerifyJob.java

```diff
@@ -127,6 +124,18 @@ static void cancel(final Context context) {
         scheduler.cancel(FIRST_RUN_JOB_ID);
     }
 
+    final String domain() {
+        return getString(R.string.url);
+    }
+
+    final String challenge_url() {
```

Comment:
Method names should also match the code/naming style of the rest of the codebase even if they're private methods

```suggestion
    final String challengeUrl() {
```